### PR TITLE
Removing {c,link}opts from grpc_cc_library

### PR DESCRIPTION
**Pull request by nicolasnoble**

### CODE REVIEW

File: bazel/grpc_build_system.bzl [hunk 1]
```diff
@@ -74,8 +74,8 @@ def grpc_cc_library(
         visibility = None,
         alwayslink = 0,
         data = [],
-        copts = [],
-        linkopts = []):
```
Comment (vjpai) [inline bazel/grpc_build_system.bzl]: Formatting issue? Loses a paren and colon?

Comment (vjpai) [reply bazel/grpc_build_system.bzl]: Rather, syntax issue, not formatting. Build broken.

Comment (nicolasnoble) [reply bazel/grpc_build_system.bzl]: Going too fast.

**Comment by nicolasnoble** [comment]
Everything has been properly reversed by other PRs.

**Pull request closed without merging**